roxie/ccd/ccdserver.cpp

```diff
@@ -17033,8 +17051,23 @@ class CRoxieServerNWayBaseActivity : public CRoxieServerMultiInputBaseActivity
 
     virtual void stop()
     {
-        ForEachItemIn(i, expandedStreams)
-            expandedStreams.item(i)->stop();
```

Comment:
expandedStreams are not set up if the activity is stopped without having been started (because it's inside an IF for example)